components/softhsm/softhsm.p5m

```diff
@@ -0,0 +1,48 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2015 Adam Stevko 
+#
+
+set name=pkg.fmri value=pkg:/security/softhsm@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value=org.opensolaris.category.2008:System/Security
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license LICENSE license='$(COMPONENT_LICENSE)'
+
+dir  path=etc
+file path=etc/softhsm2.conf
+dir  path=usr
+dir  path=usr/bin
+file path=usr/bin/softhsm2-dump-file
+file path=usr/bin/softhsm2-keyconv
+file path=usr/bin/softhsm2-util
+dir  path=usr/lib
+dir  path=usr/lib/softhsm
+file path=usr/lib/softhsm/libsofthsm2.a
+file path=usr/lib/softhsm/libsofthsm2.la
+file path=usr/lib/softhsm/libsofthsm2.so
+dir  path=usr/share
+dir  path=usr/share/man
+dir  path=usr/share/man/man1
+file path=usr/share/man/man1/softhsm2-dump-file.1
+file path=usr/share/man/man1/softhsm2-keyconv.1
+file path=usr/share/man/man1/softhsm2-util.1
+dir  path=usr/share/man/man5
+file path=usr/share/man/man5/softhsm2.conf.5
+dir  path=var
+dir  path=var/lib
+dir  path=var/lib/softhsm
+dir  path=var/lib/softhsm/tokens
```

Comment:
What is this directory used for? Seems to be a bit Linux-like to deliver something in /var/lib.